style/android/java/install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Installs Thoughtbot's IntelliJ configs into your user configs.
+
+echo "Installing Thoughtbot IntelliJ configs..."
+
+CONFIGS="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/configs"
+
+for i in $HOME/Library/Preferences/IntelliJIdea*  \
```

Comment:
It's probably a good idea to put `$HOME` in quotes: `"$HOME"/Library/Preferences/IntelliJIde`